ui/app/AppLayouts/Wallet/controls/RecipientViewDelegate.qml

```diff
@@ -0,0 +1,71 @@
+import QtQuick 2.15
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import utils 1.0
+
+StatusListItem {
+    id: root
+    required property var modelData
+
+    property bool useAddressAsLetterIdenticon: false
+
+    readonly property string name: modelData.name
```

Comment:
why are these needed? either we can make all inputs to this delegate simple properties like name, address etc and remove the modelData?